test/simpleatsit5_tests.jl

```diff
@@ -40,3 +40,42 @@ step!(deiip); step!(deiip)
 @test iip.t ≈ deiip.t atol=1e-12
 
 sol = solve(odeoop,SimpleATsit5(),dt=dt)
+
+# Test keywords:
+oop = init(odeoop,SimpleATsit5(),dt=dt, reltol = 1e-9, abstol = 1e-9)
+step!(oop); step!(oop)
+deoop = DiffEqBase.init(odeoop, Tsit5(); dt = dt, reltol=1e-9, abstol=1e-9)
+step!(deoop); step!(deoop)
+
+@test oop.u ≈ deoop.u atol=1e-14
```

Comment:
e-14 isn't stable across architectures

Reply:
Sure, I'll change this, but the problem is not with the ATsit, it is with the Tsit. This is where the error happens. And the error is big, like you need `atol = 1` to match the states...

Reply:
Interesting, I swear it was passing locally?

Reply:
I was also super confused... :( 

I've changed the tolerances to 1e-9 everywhere to account for what you said.